helpers/deployer/gcp/gcp.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcp
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/mitchellh/go-testing-interface"
+	"github.com/tidwall/gjson"
+
+	"github.com/terraform-google-modules/terraform-example-foundation/test/integration/testutils"
+)
+
+type GCP struct {
+	Runf      func(t testing.TB, cmd string, args ...interface{}) gjson.Result
+	sleepTime time.Duration
+}
+
+// NewGCP creates a new wrapper for Google Cloud Platform CLI.
+func NewGCP() GCP {
+	return GCP{
+		Runf:      gcloud.Runf,
+		sleepTime: 20,
+	}
+}
+
+// GetBuilds gets all Cloud Build builds form a project and region that satisfy the given filter.
+func (g GCP) GetBuilds(t testing.TB, projectID, region, filter string) []gjson.Result {
```

Comment:
nit: Would it be better to parse and return a map[id(string)]status(string) if that is what we need at the callsite?

Reply:
done